app/views/hyku/api/v1/collection/_collection.json.jbuilder

```diff
@@ -14,6 +14,11 @@ json.resource_type collection.resource_type&.first
 json.rights_statements_for_api_tesim collection.rights_statement&.first
 json.thumbnail_base64_string nil
 json.thumbnail_url collection.thumbnail_path
+if Hyrax::PresenterFactory.build_for(ids: [collection.thumbnail_id], presenter_class: Hyrax::FileSetPresenter, presenter_args: [current_ability, request]).first&.solr_document&.public?
```

Comment:
Could we establish a maximum line length? 120 is the norm for this and makes code much easier to read - IMO. This one is 184. 

It could be flagged as part of the rubocop configuration.

Reply:
That's a good idea.  The Samvera community style standards have it bumped up to 200 (See https://github.com/samvera-labs/bixby/blob/master/bixby_default.yml#L470).  We could lower that here but I'd like to do that in a separate PR since it looks like it affects 84 lines.  Maybe we could also lobby for changing the community standard so we don't have to override it here.

Reply:
OMG 200!? That seems incredibly long to me. 

I am a little dyslexic and longer lines are hard to parse for me. 

Id be happy to look at this PR so that we could reduce it down to a manageable level.

Reply:
I think you're experiences are a good reason to push for lowering the limit.  Do you mind writing up an issue for bixby?
https://github.com/samvera-labs/bixby/issues/new